pkg/backend/display/display.go

```diff
@@ -63,6 +64,12 @@ func ShowEvents(
 	}
 }
 
+var colorCharRegex = regexp.MustCompile(`<{%\w*\s?\d*%}>`)
+
+func cleanColorRenderingChars(s string) string {
+	return colorCharRegex.ReplaceAllString(s, "")
```

Comment:
Rather than using regexes, you can use the [`Never`](https://pkg.go.dev/github.com/pulumi/pulumi/sdk/v3@v3.12.0/go/common/diag/colors#Colorization) colorization:

```suggestion
	return colors.Never.Colorize(s)
```